apps/frontend/src/api/__tests__/use-safe-fetch.test.tsx

```diff
@@ -1,178 +0,0 @@
-import nock, { ReplyFnContext, ReplyFnResult } from 'nock';
```

Comment:
good riddance!

Reply:
Have we removed useHttp from Package.json?

Reply:
thanks!